precompiles/parachain-staking/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "parachain-staking-precompiles"
+version = "0.6.0"
```

Comment:
this version is from the precompiles crate right?

Reply:
Yeah, just a little 🖨️🍝. I would have made it 0.1.0 if I had seen it, but may as well not run the CI again.